web/themes/joinup/js/form.js

```diff
@@ -45,6 +45,9 @@
       $(context).find('form').once('ajaxReload').each(function () {
         $(document).ajaxComplete(function (event, xhr, settings) {
           componentHandler.upgradeAllRegistered();
+          $('.mdl-js-checkbox').each(function (index, element) {
```

Comment:
I think you can revert a commit. It'd be easier than rebase.
This one is related to ISAICP-3237 which is in progress still.